connectors/veilarboppfolging/src/main/kotlin/no/nav/amt/tiltak/connectors/veilarboppfolging/VeilarboppfolgingClient.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.amt.tiltak.connectors.veilarboppfolging
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import no.nav.amt.tiltak.core.port.VeilarboppfolgingConnector
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import java.util.function.Supplier
+
+class VeilarboppfolgingClient(
+	private val apiUrl: String,
+	private val tokenSupplier : Supplier<String>
+) : VeilarboppfolgingConnector {
+
+	private val objectMapper = ObjectMapper().registerKotlinModule()
+	private val httpClient = OkHttpClient()
+
+	override fun hentVeilederIdent(fnr: String) : String? {
+		val request = Request.Builder()
+			.url("$apiUrl/person/$fnr/veileder")
```

Comment:
Vi ønsker heller å bruke V2 endepunktet i veilarboppfolging. 
Vi blir også nødt til å kjøre dette requestet gjennom en proxy, så URLen kan potensielt endres litt pga det

Reply:
Åja, var ikke klar over at det er to.. Tenkte at proxyen kan komme i neste omgang siden jeg allerede startet på oppgaven

Reply:
Da er PRen min i veilarboppfølging feil også da